Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -2853,6 +2894,22 @@ public void StartThrottleIncrease(float? target)
 
         public void StartThrottleIncrease()
         {
+            if (CruiseControl != null)
```

Comment:
It would be better if you moved this to the end of StartThrottleIncrease() method, so the combined controls can be used normally, and the cruise control code only swallows the command when the combined control is in throttle positions.
The same applies for all Start/Stop Throttle Increase/Decrease methods

Reply:
Well, some locos uses the throttle handle as CruiseControl mode selector (i. e. neutral (no force), drive (force until speed reached), confirmation for drive (need to be held until given speed) when in automatic mode. Then the lever in fact is not controlling the throttle anymore, and the logic for brakes and force is completely driven by cruise control. So in that case is necessary only the cruise control to be executed and the rest is skipped completely.

Reply:
How does the throttle handle behave when the cruise control is disconnected? If it is just a single throttle lever (without combining brakes), then the changes that I suggest aren't affected. Could you please provide more accurate description of the throttle lever on these locos?

I have in mind some locomotives with a combined control lever, where the throttle acts as a speed selector, but when the lever is moved to a braking position, the brakes are applied and the cruise control is disconnected.

Reply:
Imagine you have a switch, with positions M and A. It the switch is in M position, the lever (plus/neutral/minus) works as a throttle - cruise control disconnected. If it's set to A, then the cruise control is active, but the train will not move, until you momentarily set the throttle lever to plus. The cc will then set the throttle virtually to 100%, and the force is maintained internally by overriding the UpdateMotiveForce. So the throttle is set to 100%, but the force to the loco is internally computed by cc.

Reply:
Maybe wait, until I upload the new MultiPositionController class, with wich you can stack up any controller with predefined steps. These are also bound to cruise control if that iv available. The variety of controls aroud the globe is large, so I decided to do it this way, because the normal throttle lever or combined lever were not sufficient.

Reply:
Here is a sample of such a controller:
`	ORTSMultiPositionController (
		Positions (
			Position ( ThrottleIncrease SpringLoadedBackwardsImmediatelly "Souhlas" )
			Position ( Drive CruiseControl.NeedIncreaseAfterAnyBrake "Jízda" )
			Position ( Neutral Default "Výběh" )
			Position ( DynamicBrakeIncrease SpringLoadedForwards "EDB" )
			Position ( TrainBrakeIncrease SpringLoadedForwards "Brzda" )
			Position ( EmergencyBrake Null "Rychlobrzda" )
		)
	)
`